src/main/resources/spring/migration-bean.xml

```diff
@@ -156,6 +159,10 @@
         <constructor-arg type="java.lang.String" value="index" />
     </bean>
 
+    <bean id="namespaceFile" class="java.io.File">
+        <constructor-arg type="java.lang.String" value="src/test/resources/namespaces.properties"/>
```

Comment:
So move migration-bean.xml to `test` since it's really an example, replacing it-setup.xml?  Seems like the best course of action since the production instance's app context setup must be supplied on the command line.